src/components/shared/slider/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Flex, Slider as SliderNB, Text, View} from 'native-base';
+import {
+  Control,
+  Controller,
+  FieldValues,
+  Path,
+  UnPackAsyncDefaultValues,
+} from 'react-hook-form';
+import CustomTooltip from '../../tooltip';
+import styles from './styles';
+
+interface Props<TFormValues extends FieldValues> {
+  name: Path<UnPackAsyncDefaultValues<TFormValues>>;
+  label?: string;
+  control: Control<TFormValues>;
+}
+
+const Slider = <
+  TFormValues extends {
+    [key: string]: string;
+  },
+>({
+  name,
+  label,
+  control,
+}: Props<TFormValues>) => {
+  const LIMIT_VALUES = ['0', '100'];
```

Comment:
A good idea toward the shared component logic might be to allow these values to be configurable through customProps, passing a min/max value and defaulting them to 0/100. If this approach is taken, modifying minValue and maxValue consequently is necessary.

What do you think?

Reply:
good idea